src/buildercore/bootstrap.py

```diff
@@ -447,7 +449,7 @@ def upload_master_configuration(master_stack, master_configuration_data):
 
 @updates('ec2')
 @core.requires_active_stack
-def update_ec2_stack(stackname, context, concurrency=None, formula_revisions=None, **kwargs):
+def update_ec2_stack(stackname, context, concurrency=None, formula_revisions=None):
```

Comment:
I'm becoming a bit sceptical of funcs accepting a `**splat` these days. Explicit is better than implicit and all that.